norduser/fileshare.go

```diff
@@ -68,17 +68,18 @@ func startFileshare(fileshareProcessManager *childprocess.GRPCChildProcessManage
 	case childprocess.CodeFailedToCreateUnixScoket:
 		fallthrough
 	case childprocess.CodeAlreadyRunning:
-		log.Println(internal.InfoPrefix+" fileshare started, final result: ", result)
+		log.Println(internal.InfoPrefix, "fileshare started, final result:", result)
 		return true
 	}
 
-	log.Printf(internal.ErrorPrefix+" failed to start fileshare: %d, will retry", result)
+	log.Printf(internal.ErrorPrefix+" failed to start fileshare: %d, will retry\n", result)
```

Comment:
Would be nice to have code style consistency i.e. see line #71 above

Reply:
Done :+1: